src/strategies/poap-with-weight/index.ts

```diff
@@ -0,0 +1,99 @@
+import { multicall, subgraphRequest } from '../../utils';
+import examplesFile from './examples.json';
+import aboutFile from './README.md';
+
+export const author = 'gawainb';
+export const version = '1.0.0';
+export const examples = examplesFile;
+export const about = aboutFile;
```

Comment:
This is not required anymore, which is failing the test cases, please remove this line
```suggestion
```